ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleWizard.tsx

```diff
@@ -1,11 +1,103 @@
+
+const validationSchema: yup.ObjectSchema<InitBundleWizardValues> = yup.object().shape({
+    name: yup
+        .string()
+        .trim()
+        .matches(
+            nameValidatorRegExp,
+            'Name can have only the following characters: letters, digits, period, underscore, hyphen (but no spaces)'
+        )
+        .required('Bundle name is required'),
+    installation: yup.string().trim().required(), // Select
+    platform: yup.string().trim().required(), // Radio
+});
 
 function InitBundleWizard(): ReactElement {
+    const history = useHistory();
+    const [errorMessage, setErrorMessage] = useState('');
+    const formik = useFormik({
+        initialValues,
+        onSubmit: (values, { setSubmitting }) => {
+            setSubmitting(true);
+            const { installation, name } = values;
+            generateClusterInitBundle({ name })
+                .then(({ response }) => {
+                    setErrorMessage('');
+                    downloadBundle(installation, response); // TODO try catch?
+                    setSubmitting(false);
+                    history.goBack(); // to table
+                })
+                .catch((error) => {
+                    setErrorMessage(getAxiosErrorMessage(error));
+                    setSubmitting(false);
+                });
+        },
+        validateOnMount: true, // disable Next when Name is empty
+        validationSchema,
+    });
+    const { isSubmitting, isValid, submitForm } = formik;
+
     return (
         <>
-            <InitBundlesHeader title="Create bundle" />;
+            <InitBundlesHeader title="Create bundle" />
+            <Divider component="div" />
+            <PageSection
+                variant="light"
+                isFilled
+                hasOverflowScroll
+                padding={{ default: 'noPadding' }}
+                className="pf-u-h-100"
+            >
+                <Wizard
+                    onClose={() => {
+                        history.goBack();
+                    }}
+                    onSave={submitForm}
+                    steps={[
+                        {
+                            id: 1,
+                            name: nameOfStep1,
+                            component: <InitBundleWizardStep1 formik={formik} />,
+                            enableNext: isValid,
+                        },
+                        {
+                            id: 2,
+                            name: nameOfStep2,
+                            component: (
+                                <InitBundleWizardStep2
+                                    errorMessage={errorMessage}
+                                    formik={formik}
+                                />
+                            ),
+                            nextButtonText: 'Download',
+                            canJumpTo: isValid,
+                            enableNext: isValid && !isSubmitting,
```

Comment:
i thought we were leaning towards not disabling the `next` but instead scrolling to the top with a callout of the validation error?